tools/local-compliance/loadGeneratingNodeScanner.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/compliance/collection/compliance"
+	"github.com/stackrox/rox/compliance/collection/intervals"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+// LoadGeneratingNodeScanner is a scanner that generates fake scans with high frequecy of the node-inventory messages.
+// Its main purpose is to generate load for load-testing of Sensor
+type LoadGeneratingNodeScanner struct {
+	log          *logging.Logger
```

Comment:
Why are you providing a logger as a dependency? 

I think the common pattern in the project is to define the log locally in each package like:
```
var (
     log = logging.LoggerForModule()
)
```